frontend/src/components/login/index.jsx

```diff
@@ -30,7 +30,7 @@ const Login = ({getToken}) => {
     })
   };
 
-  if(redirect) return <Redirect to="/private-settings"/>;
+  if(redirect) return <Redirect to="/change-password"/>;
```

Comment:
Why redirect to this page after login? Shouldn't we be redirected to the preference-page?

Reply:
Temporary, this Will be changed when other components such as menu and so on is built